[IOSP-514] [CNSMR-3272] Technical Documentation for Stevenson and CRP (#46)
Babylonpartners/Stevenson: README.md

# 🤖 Stevenson Bot

> `Stevenson` is a Vapor framework designed to build integrations between Slack apps, Github, JIRA and CI services (CircleCI).  

Reviewer: 👀 
```suggestion
`Stevenson` is a Vapor framework designed to build integrations between Slack apps, GitHub, JIRA and CI services (CircleCI).
```

### Supported services

> `Stevenson` comes with implementation of Slack [slash commands](https://api.slack.com/slash-commands), GitHub, JIRA and CircleCI APIs. At the moment it does not implement complete set of these APIs but only provides bare minimum required for the functionality of the app.

Reviewer: ```suggestion
`Stevenson` comes with implementation of Slack [slash commands](https://api.slack.com/slash-commands), for GitHub, JIRA and CircleCI APIs. At the moment, it does not implement complete set of these APIs but only provides bare minimum required for the functionality of the app.
```

 1. Go to the Slack Commands config page for your team's Slack app: `https://api.slack.com/apps/<YourSlackAppID>/slash-commands`
 2. Click on "Create New Command"
   * Fill in the slash command (e.g. `/foo`)

>    * Enter the URL this command will trigger – e.g. if your instance will be hosted on Heroku  it will look like `https://<appname>.herokuapp.com/<command>`, replacing `<appname>` with the name of your Heroku app instance (e.g. `stevenson-bot`) and `<command>` by the command name (e.g. `foo`)

Reviewer 2: ```suggestion
   * Enter the URL this command will trigger – e.g. if your instance will be hosted on Heroku it will look like `https://<appname>.herokuapp.com/<command>`, replacing `<appname>` with the name of your Heroku app instance (e.g. `stevenson-bot`) and `<command>` by the command name (e.g. `foo`)
```

## 📖 Documentation

> * To read the documentation about the Stevenson's instance we use at Babylon and the commands that we implemented in our usage of Stevenson, see [this page of the documentation](Documentation/BabylonCommands.md)

Reviewer 2: ```suggestion
* To read the documentation about the Stevenson's instance we use at Babylon and the commands that we implemented to support our processes, see [this page of the documentation](Documentation/BabylonCommands.md)
```
